be/src/exec/pipeline/pipeline_driver_executor.cpp

```diff
@@ -330,6 +331,8 @@ void GlobalDriverExecutor::report_exec_state(QueryContext* query_ctx, FragmentCo
         } else {
             LOG(INFO) << "[Driver] Succeed to report exec state: fragment_instance_id=" << print_id(fragment_id);
         }
+        // Force lambda capture obj_pool to take over the ownership of the object pool.
+        obj_pool->clear();
```

Comment:
My question is why this lambda needs to take ownership of obj_pool?

Reply:
the actual report action is being done in a separate thread pool.

Reply:
why does the actual report action need obj_pool?

Reply:
It's stupid, the profile object is no longer required after serialized into `params`.

Updated.